demisto_sdk/commands/validate/validators/IM_validators/IM106_default_image_validator.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.constants import (
+    DEFAULT_DBOT_IMAGE_BASE64,
+    DEFAULT_IMAGE_BASE64,
+)
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Integration
+
+
+class DefaultImageValidator(BaseValidator[ContentTypes]):
+    error_code = "IM106"
+    description = "Checks if the integration has an image other than the default ones."
+    rationale = "If an author image is provided, it must not be the default ones."
+    error_message = "This is the default image, please change to the integration image."
```

Comment:
```suggestion
    error_message = "The integration is using the default image at {0}, please change to the integration image."
```
Where 0 is the image path.